filtered_registries/simple_standard_scenarios/dump/data_path_a8380d6bc2f10f8ec17d89e5ad037c95.data.json

```diff
@@ -1 +1 @@
-{"_key":"1069275","_id":"data_path/1069275","_rev":"_g_9v5zW---","data_path":"/scratch/dthom/dsgrid/ss2/simple_standard_scenarios-data"}
+{"_key":"1069275","_id":"data_path/1069275","_rev":"_g_9v5zW---","data_path":"dsgrid-test-data/filtered_registries/simple_standard_scenarios/data"}
```

Comment:
Tested it, I think it's just `dsgrid-test-data/filtered_registries/simple_standard_scenarios/` @daniel-thom